lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -275,21 +277,24 @@ functions:
             awsprofile: default
             recemail: nci.monitor@dea.ga.gov.au # Recepient's email address
   execute_clean:
+    # Note: Keep `--search-paths` argument as the last argument in the environment.cmd
     handler: handler.execute_ssh_command
-    description: Execute dea-clean tool to clean the file on the disk
+    description: Execute dea-clean tool to trash files on disk whose index were archived at least 3 days ago
     environment:
       cmd: 'execute_clean --dea-module ${self:provider.environment.DEA_MODULE}
                           --queue ${self:provider.environment.QUEUE}
                           --project ${self:provider.environment.PROJECT}
-                          --min-trash-age-hours <%= mintrashage %>
-                          --search-string <%= searchstr %>'
+                          --min-trash-age <%= min_trash_age %>
+                          --search-paths <%= search_paths %>'
     events:
       - schedule:
-          rate: cron(00 11 ? * SUN *) # Run every Sunday, at 09:00 pm Canberra time
-          enabled: false
+          rate: cron(00 13 ? * TUE *) # Run every Tuesday, at 11:00 pm Canberra time
+          enabled: true
```

Comment:
I was very concerned about running `dea-clean` automatically. But I see that it's set to `--dry-run`. It probably should be made clear here that it's just going to list things to be cleaned, not go and automatically delete files...

Reply:
Sure, Damien. I will add a comment in the serverless file.